docs/Community/CodeOfConduct.md

```diff
@@ -0,0 +1,11 @@
+# Open Enclave SDK Code of Conduct
```

Comment:
I would like to suggest that we use one of the standard locations/filenames GitHub expects: https://help.github.com/en/github/building-a-strong-community/adding-a-code-of-conduct-to-your-project#adding-a-code-of-conduct-manually

That way it will get displayed in the Community tab, and it will probably be slightly easier to find.

Reply:
Great catch! I like it and will update. Thanks.